### ISSUE TITLE
scVelo() Error in installConda() : conda installation failed with status code '1'

### ISSUE THREAD

**Issue by shijianasdf** [kevinrue/velociraptor#47]
When I try to run the scvelo() function I get the following error message (see attached picture).

<img width="2472" alt="截屏2021-08-12 上午7 30 59" src="https://user-images.githubusercontent.com/39074222/129116651-ef471ce6-8fb1-4bef-bf04-6255836cba9f.png">

**Comment by LTLA** [comment]
You should post errors in formatted code blocks rather than taking screenshots, it's hard for me to read.

I can make out a MD5 sum mismatch, which pretty much says it all. I would guess that the download of the installer was interrupted or corrupted at some point, as discussed [here](https://stackoverflow.com/questions/61110558/why-is-anaconda-installer-failing-on-centos). Suggest going to:

```r
dirname(basilisk.utils:::getExternalDir())
```

and deleting the old installer. Then restart your R session and try running `scvelo()` again.

**Comment by shijianasdf** [comment]
> You should post errors in formatted code blocks rather than taking screenshots, it's hard for me to read.
> 
> I can make out a MD5 sum mismatch, which pretty much says it all. I would guess that the download of the installer was interrupted or corrupted at some point, as discussed [here](https://stackoverflow.com/questions/61110558/why-is-anaconda-installer-failing-on-centos). Suggest going to:
> 
> ```r
> dirname(basilisk.utils:::getExternalDir())
> ```
> 
> and deleting the old installer. Then restart your R session and try running `scvelo()` again.
After go to:
`dirname(basilisk.utils:::getExternalDir())`
I found a folder containing "Miniconda3-py37_4.8.3-MacOSX-x86_64.sh"，and I delete the folder named 'basilisk',
Then i download the package again, after run scVelo() I get the following error message 
`试开URL’https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.3-MacOSX-x86_64.sh'
Content type 'application/x-sh' length 54829425 bytes (52.3 MB)
==
downloaded 2.4 MB

Error in download.file(url, fname, mode = "wb") : 
  download from 'https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.3-MacOSX-x86_64.sh' failed
此外: Warning messages:
1: In download.file(url, fname, mode = "wb") :
  downloaded length 2538726 != reported length 54829425
2: In download.file(url, fname, mode = "wb") :
  URL 'https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.3-MacOSX-x86_64.sh': Timeout of 60 seconds was reached`
I think the network is not good and can't download it

**Comment by shijianasdf** [comment]
> You should post errors in formatted code blocks rather than taking screenshots, it's hard for me to read.
> 
> I can make out a MD5 sum mismatch, which pretty much says it all. I would guess that the download of the installer was interrupted or corrupted at some point, as discussed [here](https://stackoverflow.com/questions/61110558/why-is-anaconda-installer-failing-on-centos). Suggest going to:
> 
> ```r
> dirname(basilisk.utils:::getExternalDir())
> ```
> 
> and deleting the old installer. Then restart your R session and try running `scvelo()` again.

Thanks for your advice.
After go to:
> 
> ```r
> dirname(basilisk.utils:::getExternalDir())
> ```
I found a folder containing "Miniconda3-py37_4.8.3-MacOSX-x86_64.sh" and i delete the folder.
then i download the package again, I run scVelo() again and got  the following error message:
>  ```r
>试开URL’https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.3-MacOSX-x86_64.sh'
>Error in download.file(url, fname, mode = "wb") : 
>  无法打开URL'https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.3-MacOSX-x86_64.sh'
>此外: Warning message:
>In download.file(url, fname, mode = "wb") :
>  URL 'https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.3-MacOSX-x86_64.sh': status was 'Couldn't resolve host >name'
>```
I think the network is bad and the files are not downloaded

**Comment by LTLA** [comment]
Yes, that would also be my conclusion. You could try downloading the installer manually and sticking the file in that directory, hopefully it should trick **basilisk** into using it.
